src/harpstrata/Pitch/pitchMap/index.test.ts

```diff
@@ -0,0 +1,32 @@
+import { PitchIds } from '../types'
+import type { Pitch } from '../types'
+
+import { getOrderedPitchIds, getPitch } from './index'
+
+
+test('getOrderedPitchIds function returns an ordered array of the available pitches defaulting to starting at C', () => {
```

Comment:
I'm should consider removing all of the default functionality. I'm not sure I want to provide variability like that.

On the other hand it's a totally optional feature. The consumer doesn't need to know about it to be able to make sense of the function.

Reply:
https://github.com/js-jslog/harpstrata/issues/18 raised.